docs/05-concepts/06-database/10-raw-access.md

```diff
@@ -1,45 +1,80 @@
-# Raw access
+# Raw Access
 
-Sometimes more advanced tasks need to be performed on the database. For those occasions, it's possible to run raw SQL queries on the database.
+The library provides methods to execute raw SQL queries directly on the database for advanced scenarios.
 
-Use the `unsafeQuery` method when querying for simple data. The method returns a `List<List<dynamic>>` with rows and columns and is useful if you want a simple result without joining any data.
+## `unsafeQuery`
+
+Executes a single SQL query and returns a `DatabaseResult` containing the results. This method uses the extended query protocol, allowing for parameter binding to prevent SQL injection.
 
 ```dart
-PostgreSQLResult result = await session.db.unsafeQuery(
-  'SELECT * FROM mytable WHERE id = 1',
+DatabaseResult result = await session.db.unsafeQuery(
+  r'SELECT * FROM mytable WHERE id = @id', 
+  parameters: QueryParameters.named({'id': 1}),
 );
 ```
 
-Use the `unsafeQueryMappedResults` method when making complex queries with joins. The method returns a `List<Map<String, Map<String, dynamic>>>`, where the `List` contains an entry for each result row. The result row is a `Map` where the key is the table from which the data has been retrieved and the value is another `Map` of that tables columns. The column `Map` inside the table `Map` has column names as keys and the contents of the colum as values.
+## `unsafeExecute`
+
+Executes a single SQL query without returning any results. Use this for statements that modify data, such as `INSERT`, `UPDATE`, or `DELETE`. Returns the number of rows affected.
```

Comment:
Suggestion: Change `without returning any results` as the method returns the number of effected rows.